packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2462,3 +2464,353 @@ Specific property types support a subset of [JSON Schema validations](https://to
 | `integer`     | [JSON Number](https://tools.ietf.org/html/rfc7159#section-6) with 32-bit signed two's complement integer constraint           | `minimum` and `maximum`     |
 | `array`       | [JSON Array](https://tools.ietf.org/html/rfc7159#section-5)                                                                         |                             |
+
+## Unique attribute
+
+<ApiLifecycle access="ea" />
+
+You can enforce uniqueness for custom properties in the Okta user profile or the Okta group profile, such as an employee identification number. You can declare a maximum of five unique properties for each user type and five unique properties in the Okta group profile. Different user types can have the same or different unique properties (up to the limit of five per type).
+
+Unique properties in the Okta user profile share a single namespace across all [user types](/docs/reference/api/user-types) in an org. If user types A and B both contain the property `ice cream` and you identify it as unique in both profiles, then if user type A has the value `chocolate`, no other users of type A or B (or any other user type with `ice cream` declared unique) can have that value.
+
+Properties that aren't unique aren't tracked for uniqueness. If the property `candy` is unique in type E and not unique in type F, and a user of type E has the value `caramel` for the property, then no other users of type E can have the value `caramel` for the property. But any number of users of type F can have the value `caramel`. Although `candy` is unique in E, it isn't unique in F, so the value for the property in users of type F doesn't matter.
+
+If you attempt to create or update a user with a duplicate value for a custom user property with a uniqueness restriction, the user creation or update operation fails. The user isn't created or updated until you enter a unique value. Similarly, you need to provide a unique value for a unique custom group property to create or update a group.
+
+`null` value doesn't come into the uniqueness calculation. If the unique property is not also specified as being required, you can also omit the value entirely; multiple users or groups can omit the property and not violate uniqueness.
+
+To enforce uniqueness for custom properties, you can either add new unique custom properties or update existing custom properties to be unique.
+
+### Add new unique custom property
+
+You can use the [add property to user profile schema](#add-property-to-user-profile-schema) request or the [add property to group profile schema](#add-property-to-group-profile-schema) request to add one or more unique custom user or group properties. Specify `"unique": true` on the properties that need to be marked as unique. After the request is submitted, the property's uniqueness status is changed immediately to `"unique": "PENDING_UNIQUENESS"` immediately and uniqueness is then enforced on that property.
```

Comment:
This is wrong.  For a new property it goes immediately to UNIQUE_VALIDATED.  And would it be correct to say the status is changed "when" the request is submitted, not "after"?  After all, the response to the request already shows it as validated, right?  So (note I also edited out the words "that need"):

Specify `"unique": true` on the properties to be marked as unique. The response shows the property with `"unique": "UNIQUE_VALIDATED"` and uniqueness is then enforced on that property.

Reply:
oops sorry the status was a copy-paste mistake.

Reply:
Should the later part of the sentence use plural since the first half mentions `properties`? Updated to:
```
Specify `"unique": true` on the properties to be marked as unique. The response shows the properties with `"unique": "UNIQUE_VALIDATED"` and uniqueness is then enforced on those properties.
```